sdks/python/apache_beam/runners/common.py

```diff
@@ -1086,6 +1053,9 @@ def _invoke_process_batch_per_window(
 
     kwargs_for_process_batch = kwargs_for_process_batch or {}
 
+    if additional_kwargs:
```

Comment:
Is this a drive-by change or is it directly related to the intention of this PR?